colaraz/lms/templates/courseware/progress.html

```diff
@@ -55,11 +55,22 @@
                 lang="${course.language}"
               % endif
               >
-                % if staff_access and studio_url is not None:
-                <div class="wrap-instructor-info">
-                    <a class="instructor-info-action studio-view" href="${studio_url}">${_("View grading in studio")}</a>
-                </div>
-                % endif
+                <header class="page-info-header">
+                    <div class="image-frame">
+                        <img src="${static.url('images/course.jpg')}" alt="Autodesk Maya">
+                    </div>
+                    <div class="content-area">
+                        <div class="form-actions">
+                            % if staff_access and studio_url is not None:
+                            <div class="wrap-instructor-info">
+                                <a class="instructor-info-action studio-view" href="${studio_url}">${_("View grading in studio")}</a>
+                            </div>
+                            % endif
+                        </div>
+                        <h2>Course title</h2>
```

Comment:
We can ignore this although I guess.
```suggestion
                        <h2>${_("Course title"}</h2>
```

Reply:
It is just a static string put for a developer, who will update it.